### PR TITLE
Tag field to look similar to list field

### PR DESCRIPTION
**Pull request by fonsekaean**
This contains bits of CSS and JS updates so that the tag fields looks similar to the ListBox fields and other fields in the CMS.

### CODE REVIEW

**Comment by andrewandante** [comment]
@fonsekaean You'll need to use the correct class for `ReadonlyTagField` - and a couple of screenshots to show the change would be great :)

File: src/TagField.php [hunk 1]
```diff
@@ -446,7 +450,7 @@ public function validate($validator)
      */
     public function performReadonlyTransformation()
     {
-        $copy = $this->castedCopy(ReadonlyTagField::class);
```
Comment (andrewandante) [inline src/TagField.php]: This is the correct class, please rebase or undo this change

Comment (fonsekaean) [reply src/TagField.php]: Yup thats fixed ;)

**Comment by fonsekaean** [comment]
Here is a screenshot of the cnage and how it looks like  
screen shot 2018-02-19 at 6 30 09 pm

**Comment by andrewandante** [comment]
Tests need fixing, but screenshot looks good, thanks!

**Comment by fonsekaean** [comment]
@andrewandante Yup those Travis issues are fixed

**Comment by andrewandante** [comment]
Excellent, happy to ignore the codecov failures. Can you rebase/squash down to one commit please, and then I'll get someone with merge-powers to pull it in?

**Comment by dhensby** [comment]
squash and merge feature is a life saver
